src/mlt++/Makefile

```diff
@@ -10,7 +10,7 @@ LIBFLAGS += -install_name $(libdir)/$(SONAME) -current_version $(version) -compa
 else ifeq ($(targetos), MinGW)
 NAME = libmlt++$(LIBSUF)
 TARGET = libmlt++-$(soversion)$(LIBSUF)
-CXXFLAGS += -DMLTPP_EXPORTS
+CXXFLAGS += -DMLTPP_EXPORTS --std=c++11
```

Comment:
Travis CI failed. Here is the way we specify c++11 in src/modules/qt/Makefile. We should do it the same way:
https://github.com/mltframework/mlt/blob/master/src/modules/qt/Makefile#L46

Or, possibly put it in into the top level configure that generates the config.mak that all Makefiles include. Then, we can safely, gradually migrate all C++ code to c++11.